elliottlib/container.py

```diff
@@ -1,62 +0,0 @@
-import shlex
-import exectools
-import logutil
-import constants
-
-logger = logutil.getLogger(__name__)
-
-
-class DockerContainer(object):
```

Comment:
This _might_ still prove useful for what we were doing with it before (basically, starting containers to run orphan checks). But that's no reason to keep this around, we can resurrect it from git if needed.

Reply:
Sounds good.